detection_rules/devtools.py

```diff
@@ -79,9 +79,10 @@ def update_lock_versions(rule_ids):
 
 @dev_group.command('kibana-diff')
 @click.option('--rule-id', '-r', multiple=True, help='Optionally specify rule ID')
+@click.option('--repo', default='elastic', help='Repository where branch is located')
```

Comment:
I think `elastic/kibana` is a better default and more accurate. Just because technically `elastic` isn't the repo but the org